services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.db import transaction
+
+from db.models import Ticket, Order, User, MovieSession
+
+
+def create_order(tickets: list,
+                 username: str,
+                 date: datetime = None) -> Ticket:
+    with transaction.atomic():
+        ticket_objects = []
+        user, user_created = User.objects.get_or_create(username=username)
+        order, order_created = Order.objects.get_or_create(user=user)
```

Comment:
Using `get_or_create` for `User` and `Order` is efficient, but ensure that this behavior aligns with your business logic, especially regarding user creation.